webpack.config.js

```diff
@@ -179,31 +172,20 @@ const config = (module.exports = {
       outputPath: __dirname + "/resources/frontend_client/app/dist",
     }),
     new webpack.DefinePlugin({
-      "process.env": { NODE_ENV: JSON.stringify(NODE_ENV) },
+      process: { env: { NODE_ENV: JSON.stringify(NODE_ENV) } },
       INCLUDE_EE_PLUGINS: JSON.stringify(process.env.MB_EDITION === "ee"),
```

Comment:
I've just searched through the entire code base and we don't use this anywhere. Probably safe to remove the whole line.